www/js/diary/infinite_scroll_list.js

```diff
@@ -109,7 +109,11 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         Logger.log("Received batch of size "+ctList.length);
         ctList.forEach($scope.populateBasicClasses);
         ctList.forEach((trip, tIndex) => {
-            trip.nextTrip = ctList[tIndex+1];
+            if (tIndex == 0) {
+              trip.prevTrip = {display_date: trip.display_date};
+            } else {
+              trip.prevTrip = ctList[tIndex-1];
+            }
```

Comment:
I can see adding `prevTrip`, but I am not sure why you want to remove `nextTrip`.
`nextTrip` is set for a reason - it is used in the user input matching code.

Reply:
Given the amount of time that this change has been pending, I will just fix this as part of the merge.